usage/rest-client/src/main/java/brooklyn/util/http/BuiltResponsePreservingError.java

```diff
@@ -62,7 +64,8 @@ public BuiltResponsePreservingError(int status, Headers<Object> headers, Object
             Exceptions.propagateIfFatal(e);
             return new BuiltResponsePreservingError(status, headers, entity, new Annotation[0], e);
         } finally {
-            source.close();
+            if (source instanceof BaseClientResponse)
```

Comment:
@aledsage the thing is, there *isn't* a `close()` method on `Response` - at least not officially. There are multiple different versions of `Response` on the classpath and sometimes the "wrong" one (one without a `close()` method) ends up at the front. See the Jira issue for full details.